python/huggingfaceserver/huggingfaceserver/__main__.py

```diff
@@ -52,10 +52,12 @@ def list_of_strings(arg):
 parser.add_argument(
     "--model_dir",
     required=False,
-    default=None,
+    default="/mnt/models",
```

Comment:
Wouldn't this be problematic as [L136](https://github.com/kserve/kserve/pull/3680/files#diff-2ed97434a3b43aac01ea6d9f2364d6c176a3716fcff260115226a61260d460f9L133)/[L156](https://github.com/kserve/kserve/blob/c660972ea153a197a483efb23b5c5e036eb4504c/python/huggingfaceserver/huggingfaceserver/__main__.py#L156) would always set args.model to `/mnt/models`?
Should the line be then updated to `args.model = args.model_id or args.model_dir`